app/helpers/application_helper.rb

```diff
@@ -174,7 +174,7 @@ def community_emoji
   end
 
   def release_adjusted_cache_key(path)
-    release_footprint = ApplicationConfig["RELEASE_FOOTPRINT"]
+    release_footprint = ForemInstance.deployed_at
```

Comment:
Using the newly created ForemInstance.deployed_at method here as will set ApplicationConfig["RELEASE_FOOTPRINT"] if it is present.